packages/amplify-category-auth/resources/adminAuth/admin-auth-package.json

```diff
@@ -10,6 +10,6 @@
     "express": "^4.17.1"
   },
   "devDependencies": {
-    "aws-sdk": "^2.1113.0"
+    "aws-sdk": "^2.1233.0"
```

Comment:
This is a resource file for Admin Queries so it doesn't need to be bumped

Reply:
reverting this change on next update